src/vidible-player.js

```diff
@@ -179,7 +211,11 @@
                                 },
                                 isMuted: function () {
                                     try {
-                                        return vidiblePlayer.getPlayerInfo().volume === 0;
+                                        var volume = vidiblePlayer.getPlayerInfo().volume;
+                                        if(volume === undefined || volume == null)
+                                            return undefined;
+                                        else
+                                            return vidiblePlayer.getPlayerInfo().volume === 0;
                                     } catch (e) {
```

Comment:
Remove the try and catch. if they want to dirt the console let them, I don't want to swallow exceptions in case they will change the implementation in the future.